dynamic-floating-group.lisp

```diff
@@ -0,0 +1,397 @@
+;; Copyright (C) Jin-Cheng Guu
+;;
+;;  This file is part of stumpwm.
+;;
+;; stumpwm is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation; either version 2, or (at your option)
+;; any later version.
+
+;; stumpwm is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this software; see the file COPYING.  If not, see
+;; <http://www.gnu.org/licenses/>.
+
+(in-package :stumpwm-dfg)
```

Comment:
To do this I should rename some symbols like `*default-layouts*` in the package `stumpwm-dfg`. I will probably do this after other things are done.

Reply:
We can keep the code in a separate package then. What do you think about importin the internal functions we are using from the stumpwm package to avoid the `stumpwm::` clutter?

Reply:
Are you sure? I am happy to leave it in a separate package too. Perhaps a few years later when people find it stable, we can merge them back anyways. It shouldn't be hard.

About the `stumpwm::` clutter: I think it's ok to have them there. I have a few `stumpwm:` anyways, and having an extra `:` doesn't look like much more clutter to me. 

Let's leave it as is, wait for a while, and decide then if merging is needed? At this moment, leaving them separated makes it easier to debug IMO. Though I have been using only it for the past 2~3 months without issues. Please let me know!

Reply:
I decide to leave it as is. Let's consider merging the package later in the future.